test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1125,9 +1127,10 @@ UCS_TEST_P(test_ucp_wireup_unified, select_best_ifaces)
 
     // Set some big enough number of endpoints for DC to be more performance
     // efficient than RC. Now check that DC is selected over RC.
-    modify_config("NUM_EPS", "1000");
-    entity *e = create_entity();
-    check_unified_ifaces(e, "dc_mlx5", "rc_mlx5");
+    // TODO: enable test when keepalive feature is enabled for DC transport
+    //modify_config("NUM_EPS", "1000");
+    //entity *e = create_entity();
+    //check_unified_ifaces(e, "dc_mlx5", "rc_mlx5");
```

Comment:
maybe exclude keepalive flag from "better transport" logic in UCP?

Reply:
as for me - this is good solution... but @brminich has concern about this...

Reply:
In this case we can close rc_x (which has keepalive) leaving just dc. Then we will not be able to support certain features (proper ep error handling) even though we have transport with needed caps.

Reply:
assert that dc does not have keepalive so test will fail if we add dc keepalive but still disable this test